### DISCUSSION TITLE
Store plane controls and commands in replay

### DISCUSSION THREAD

**Discussion by fayak** [nguyenquyhy/Flight-Recorder#9] [Ideas]
While watching the recording, I noticed that the commands and switches inside the cabin are not recorded (autopilot configuration, flaps, throttle, ...).

Is it data that is fetchable easily ? Can it be recorded and replayed as well ?

I haven't looked in depth into the flightconnect API so maybe this enhancement feature is unfeasible or too complicated.

Cheers

**Comment by nguyenquyhy** [comment]
It really depends on exactly which switches you are looking at. Currently the tool uses SimConnect, which is an interface for external tools to access sim data. It doesn't require putting anything in Community folder, but it doesn't expose all data, especially those that are very aircraft-specific. Extracting more data on the switches would require building a mod in Community folder, and even so, some data is stored inside the panels and pretty much impossible to extract without modding the panels themselves.

Besides, I don't record all SimConnect variables either. I try to make the tool more for education and some video recording purposes rather than reproduce all aspect of the sim. This is mainly to make sure I can balance the record file size and the usefulness of the data.

If you have anything specific that you think would be useful for certain purposes, please give more details and I'll try to add them in.
